api/tests/test_user.py

```diff
@@ -336,7 +363,7 @@ def test_allocated_assets_are_returned(self, mock_verify_token):
             HTTP_AUTHORIZATION="Token {}".format(self.token_admin)
         )
         count = len(response.data['allocated_assets'])
-        self.assertEqual(count, 0)
+        self.assertEqual(count, 1)
```

Comment:
This assert statement was initially validating that the count will always be zero for users who do not have an asset allocated to them. To ensure that we keep that check, you could assign `self.asset` to a different user that's not `self.user`. Alternatively, you could move the allocation on [line 47](https://github.com/AndelaOSP/art-backend/pull/171/files#diff-a88fe99b551eef0f172e7505561b4951R47) to within `test_admin_filter_users_by_asset_count()`, where you're validating the filtering by asset count.

Reply:
Prefer the latter.